src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -0,0 +1,269 @@
+/* Copyright © 2018 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#include <toolkits/style_transfer/style_transfer.hpp>
```

Comment:
<img width="1038" alt="Screen Shot 2019-10-18 at 11 40 40 AM" src="https://user-images.githubusercontent.com/4616962/67119498-233e7a00-f19c-11e9-96dd-46b1a396d28a.png">

In this instance: #include <toolkits/style_transfer/style_transfer.hpp> acts as (1. dir2/foo2.h). Rather than a relative import we're using an absolute import for this.